### ISSUE TITLE
Unmarked featured artist maps

### ISSUE THREAD

**Issue by NoffyNoffeh** [ppy/osu-web#8111]
It's been a week and from what I recall, we were asked to wait a week before reporting on unmarked FA maps. So here's a list?

[google sheet](https://docs.google.com/spreadsheets/d/1E2WlolDfcrkVgRe66Vrf6NY0m5e6RBA53UDbth8H0QA/edit?usp=sharing)

This list covers 499 ranked and loved maps of featured artist tracks which are not marked as featured artist on the website. I arbitrarily stopped at 499, this list is not fully comprehensive of all unmarked maps. I estimate over a third of all Featured Artist maps of licensed tracks are unmarked.

For tracks with multiple FAs I list only the first. Not sure how this is handled otherwise.

Common issues:
- Featured artist listing has a / between aliases. The map states only the alias which the song was released under in its artist field. The map ends up unmarked.
- There is minor formatting differences.
- (Cut Ver.) maps are not marked
- Featured artist collabs with another artist, such as with a feat. or other indicative symbol in the artist field. These maps end up unmarked.
- Markers for length versions, remixes, etc, are usually in light gray text on the FA listing, but are given as (marker) or similar in the template. Maps of these tracks with (marker) do not get marked.
- Group FAs such as High Tea, MOtOLOiD, Rising Sun Traxx, Tiny Waves, are completely unmarked due to the actual artists being completely different from the FA name.
- The FA listing and template had errors which were previously fixed in the template, but not on the listing. Or they were not fixed at all, while the ranked/loved map is correct.
- The track on the FA listing has its title in Japanese, such as *namirin and Trial & Error. Not sure why this leads to unmarked maps.
- The ranked/loved map has incorrect metadata (usually maps from before an artist was licensed.)

There is easily another 500 more or more of these, improvement is definitely needed in this area. I'm not sure what needs to be done here as the criteria for maps being marked is not publicized. Even when improvements are made, there needs to be a central location to be able to report unmarked maps so they can be fixed and linked to the Featured Artist listing, as with so many tracks and so many possible tiny variations, it seems impossible to cover every possible case automatically.

**Comment by peppy** [comment]
Featured artist beatmaps are sourced from ground truth .osz files. Metadata should match them. As long as this matches there should be no issue.

Please keep reports to issues with the templates or listings.

**Comment by NoffyNoffeh** [comment]
The osz files? That doesn't seem right, dozens of items in the list already match the FA osz files but aren't marked. See for example the Aitsuki Nakuru maps or the Loki / Thaehan maps.

**Comment by peppy** [comment]
Let me try again:

Please only report issues with the listing or templates *rather than beatmaps*. These are not yet being used, but that is considered the source of ground truth which will be used going forward.

**Comment by peppy** [comment]
An initial pass has been made to catch the more common issues. I haven't gone through your spreadsheet yet, but it'd be appreciated if you can update it as above (listing any FA-side issues) and maybe a bullet point list of any remaining common failure cases (with one beatmap example max). No need for huge lists.

**Comment by NoffyNoffeh** [comment]
- Emille's Moonlight Serenade maps match the osz but aren't getting marked. May be due to the FA name (Emille's Moonlight Serenade) and osz name (Emiru no Aishita Tsukiyo ni Dai San Gensou Kyoku wo) being different and the FA having no collabs and no featured singers. Template oszs and ranked maps match and are correct. Similar cases are popping up for other solo FAs which have a few songs with different aliases.
_[Example 1](https://osu.ppy.sh/beatmapsets/1454083#taiko/2990547), [Example 2](https://osu.ppy.sh/beatmapsets/1486362#osu/3047709)_

- Matching seems to be case sensitive when it doesn't need to be. See [Kuba Oms maps](https://osu.ppy.sh/beatmapsets?q=Kuba%20Oms) or https://osu.ppy.sh/beatmapsets/21178#osu/86531 vs. https://osu.ppy.sh/beatmapsets/757813#osu/1596819  (Cranky changed his alias, old map follows old lowercase alias). Ranked Kuba Oms maps correctly follow sources from Kuba Oms so the template osz files should be updated regardless, but being case-sensitive still doesn't seem neccessary here. 


- Other maps aren't being picked up despite matching osz, looks like in a lot of cases they have special characters in the title or artist.
Examples: https://osu.ppy.sh/beatmapsets/1393933#fruits/2882490, https://osu.ppy.sh/beatmapsets/955671#osu/1997225

Full list of maps found so far which match the template and are unmarked: [reference](https://docs.google.com/document/d/1ZyIbT93EwzUd5aKcthQfkHKMAZ6ThWPxVi7Xun9xmxA/edit?usp=sharing) _I'm mainly linking this because the reasons why they're unmarked is just guess work and would need further investigation._.


I'm working with eiri- to compile FA osz template issues for later, but these are some common cases with already matching map and template which were noticed.

**Comment by peppy** [comment]
Ah, case sensitivity was one on my list I completely forgot to apply. That's added an additional mappings.

The metadata issues you point out so fare are going to be resolved in a pass we need to do of some older artists where osz don't match databased metadata.

**Comment by peppy** [comment]
I've added some better handling when .osz doesn't match database, should have caught a lot of the edge cases you mentioned above. 37 new tracks matched.

**Comment by Hiviexd** [comment]
hi, I found a false positive that's related to the FA tag: https://osu.ppy.sh/beatmapsets/373414#taiko/818079

pretty sure toby fox isn't a featured artist, and clicking the tag sends you to Camellia's FA page

**Comment by peppy** [comment]
Thanks, will add a rule against matching on remixes like that.

**Comment by peppy** [comment]
All of the issues mentioned in https://github.com/ppy/osu-web/issues/8111#issuecomment-917548467 have been resolved. Let me know if you find any more.

**Comment by peppy** [comment]
Thanks, I've unmarked that beatmap and adjusted the rules to better handle remixes (again).

**Comment by JBHyperion** [comment]
Found a couple of missing MIMI sets for "Nanimo Nai Youna". Based on beatmapset discussion, seems to be a romanization error on the osz/listing, which reads "Nani mo Nai Youna" (with an additional space): https://osu.ppy.sh/beatmapsets/1151466/discussion/-/generalAll#/1557587

https://osu.ppy.sh/beatmapsets/1151466
https://osu.ppy.sh/beatmapsets/1197461

**Comment by Nivalyx** [comment]
@peppy would like to report another case of false positive(s) regarding the song "Yuyoyuppe - Palette" : 

---

1) Yuyoyuppe feat. Hanatan - Palette (http://osu.ppy.sh/s/15951)

this version of the song is an utaite cover of the original vocaloid version (which is licensed), yet somehow it is still tagged as "Featured Artist" in the website. not sure if this is intended behaviour or not tho

---

2) Yuyoyuppe - Palette GAMMA (http://osu.ppy.sh/s/409429 / http://osu.ppy.sh/s/679239)

...yeah, i'm pretty sure this one is just a (fanmade?) speed up edit of the song and by any means not official ;~;

**Comment by LittleEndu** [comment]
None of the maps that appear in rejection's news post are actually marked as featured artist maps. All are of the same song, "Signal feat. Such" which appears to be licensed:
 * https://osu.ppy.sh/s/1286349
 * https://osu.ppy.sh/s/1241769
 * https://osu.ppy.sh/s/1104277
 * https://osu.ppy.sh/s/1129636
 * https://osu.ppy.sh/s/1190233
 
 Also, can we get confirmation that the detection is run automatically after a new artist is announced? Like how long should we wait to report issues with new artists?

**Comment by peppy** [comment]
You should not report issues for new artists. That's out of scope of this thread. Please read my previous comments on what should be posted in this thread.

**Comment by NoffyNoffeh** [comment]
In addition to the ones Niva mentioned, for ones that are falsely flagged or unflagged

1. Other false flags
https://osu.ppy.sh/beatmapsets/847415#osu/2177436 the version with this featured singer is not licensed
https://osu.ppy.sh/beatmapsets/1350334#taiko/2818161 this version is a remaster (indicated by the title changed to remind) and is not licensed.
https://osu.ppy.sh/beatmapsets/1388590#osu/2867851 this is a remix of the licensed song, it itself is not licensed


2. Unflagged
https://osu.ppy.sh/beatmapsets/1388993#osu/2868551 matches template osz but not tagged for some reason?
https://osu.ppy.sh/beatmapsets/632589 both songs in the map are licensed, they were originally released together https://www.youtube.com/watch?v=E1JcjRRJMEo but for some reason the featured artist listing split them up into two solo tracks. Maybe a third osz with them together like original release would be a solution? Or some way to account for compilations in general, they do happen, especially with metal artists having part 1 / part 2 songs.

3. There's also quite a few old ARForest maps which list only the singer as the artist, I wouldn't say the osz templates are wrong for saying ARForest feat. Singer, but the maps weren't wrong back then either. Some way to match for just singer in these cases would be helpful.

https://osu.ppy.sh/beatmapsets/12731#osu/47535
https://osu.ppy.sh/beatmapsets/53519#osu/163054
https://osu.ppy.sh/beatmapsets/91606#osu/248001
https://osu.ppy.sh/beatmapsets/39056#osu/124771
https://osu.ppy.sh/beatmapsets/46223#osu/143989

I'm not sure of the current methodology but I think it could work like

- Check if it matches osz
- Specifcally check for (Short Ver.) / (Cut Ver.) and match if those are the only difference
- For some tracks, have set aliases it could match to besides the osz. Similar could be done for artists. 
e.g. Template has `Someone feat. Somebody` - `Title`. It could match to maps if they have either `Someone`, `Somebody`, or `Someone feat. Somebody` but NOT `Someone feat. Anotherperson`

It seems like current matching still misses quite a few things and also picks up other remixes and alternate vocal versions incorrectly.

**Comment by peppy** [comment]
No fix has been applied yet for issues I have not replied to as resolved. Please be patient. No need to attempt to define how the process works as I have that under control.

**Comment by h3oCharles** [comment]
couple songs didn't get marked that i mapped:

united laos remix https://osu.ppy.sh/beatmapsets/1323369#taiko/2741231

algebra https://osu.ppy.sh/beatmapsets/1285922#taiko/2670153
algebra again but unfinished set https://osu.ppy.sh/beatmapsets/758593#taiko/1595602

**Comment by peppy** [comment]
Graveyard is out of scope right now.

**Comment by TheMoMan** [comment]
Another false positive same song as https://github.com/ppy/osu-web/issues/8111#issuecomment-917663262

https://osu.ppy.sh/beatmapsets/750458#osu/1580331

**Comment by peppy** [comment]
> @peppy would like to report some potential cases of false positives that i found regarding the song "Yuyoyuppe - Palette" :
> 
> 1. Yuyoyuppe feat. Hanatan - Palette (http://osu.ppy.sh/s/15951)
> 
> this version of the song is an utaite cover of the original vocaloid version (which is licensed), yet somehow it is still tagged as "Featured Artist" in the website. not sure if this is intended behaviour or not tho
> 
> 1. Yuyoyuppe - Palette GAMMA (http://osu.ppy.sh/s/409429 / http://osu.ppy.sh/s/679239)
> 
> ...yeah, i'm pretty sure this one is just a (fanmade?) speed up edit of the song of some sort and by any means not official ;~;

I honestly think these kinds of edge cases are fine to leave as is. The artist is featured and the tracks are the correct (base) tracks. The main purpose here is providing visibility, not a perfect 1:1 match. So I'm going to leave these for now.

**Comment by peppy** [comment]
I've done another update covering hopefully all the issues posted above. Please resurface any I may have missed!

**Comment by h3oCharles** [comment]
found another map that isnt marked https://osu.ppy.sh/beatmapsets/875916#osu/2357967

**Comment by peppy** [comment]
Thanks, that's an edge case that wasn't handled. Will look into it tonight.

**Comment by Nivalyx** [comment]
found yet another map that are missing the marker : http://osu.ppy.sh/beatmapsets/1522511#osu/3121000

this one's possibly unmarked because to the (very slight) metadata difference between the beatmap and the fa listing page :

![image](https://user-images.githubusercontent.com/36564236/142039101-0677e906-c5cd-4305-a54a-21a9c9f3896f.png)

![image](https://user-images.githubusercontent.com/36564236/142039135-5e982997-fe7f-429e-a55b-87e49fb612df.png)

**Comment by Heaxys** [comment]
Hi, this map of mine has still not been marked until now https://osu.ppy.sh/beatmapsets/1296171
![image](https://user-images.githubusercontent.com/45066075/144527240-667ebcae-4a7f-472c-bc63-bad51f9e4a05.png)

I don't know why tho, formatting is the same, and it got submitted after the band (and the song) got listed

**Comment by AncuL001** [comment]
https://osu.ppy.sh/beatmapsets/1655287#mania/3380199 false positive

**Comment by h3oCharles** [comment]
found more maps with questionable markings
all are MIMI songs
should umu's cover of Ai no Sukima be marked?
marked ctb set https://osu.ppy.sh/beatmapsets/1215878#fruits/2530118
sotarks cut https://osu.ppy.sh/beatmapsets/966339#osu/2031676
full ver with sb https://osu.ppy.sh/beatmapsets/1371996#osu/3102474

should Nuvolina's Rock Cover of Ai no Sukima be marked too?
std set https://osu.ppy.sh/beatmapsets/1092468#osu/2283635
taiko set https://osu.ppy.sh/beatmapsets/1096932#taiko/2295515

**Comment by peppy** [comment]
Thanks for the reports.

While they aren't 100% correct, I'm not sure these edge cases matter too much. They are harder ones to automate because the titles (like "feat. someone") for a cover are not standard - this is most commonly used by an artist for an original track title where the singer wants to be recognised, not a cover.

At the end of the day it does link back to the "original" track, which allows for clarification if a user cares to check.

I guess it does depend on how we use the featured artist badge going forward, and whether having these maps marked as such leads to further confusion.

**Comment by aryoadhi** [comment]
Thanks for mentioning my issue. As stated on my topic above, there is a chance that a map is falsely marked with Featured Artist listing.

https://osu.ppy.sh/beatmapsets/1667108

Perhaps it's due to the artist involved in remixing uses the template that might conflicts with the webpage's automatic marking system, and my decision to cut the song shortly. The title is:

```
Getty - Pump It (Serji Remix) (Cut Ver.)
```

The original source is indeed available on the Featured Artist listing, but this particular remix doesn't. Perhaps it's conflicting too with the suffix `(Cut Ver.)` attached on my beatmap.

**Comment by aceredshirt13** [comment]
Hello! I know this thread is mostly dead, and several of these may have been mentioned here before, but I was told to come here by the NAT when I contacted them about this. The following maps:

https://osu.ppy.sh/beatmapsets/1251607#taiko/2601734
https://osu.ppy.sh/beatmapsets/695743#osu/1554636
https://osu.ppy.sh/beatmapsets/58951#osu/243848
https://osu.ppy.sh/beatmapsets/100016#osu/265590
https://osu.ppy.sh/beatmapsets/114480#osu/296411
https://osu.ppy.sh/beatmapsets/1108252#osu/2327672
https://osu.ppy.sh/beatmapsets/28145#osu/93957

all need to be marked Featured Artist. They all have the aforementioned issues of slight differences in how either the title of the song or the name of the artist is marked, even though they are identical to the songs listed as featured.

**Comment by peppy** [comment]
@Ephemeralis I think you already looked into this one – did you happen to have a fix lined up? I think you said the beatmap template was actually incorrect.

**Comment by Ephemeralis** [comment]
I can go through and link them to tracks manually via DB if that won't break anything. This is typically caused by a metadata mismatch - either the beatmapset's metadata is incorrect (likely the case on older ones) or the template on our listing is incorrect (which it shouldn't be and warrants an update to fix).

**Comment by peppy** [comment]
I mean, depends on which is the case here. Either way, you can make changes at a database level but let me know when you're done as I will need to run reindexing.

**Comment by Wieku** [comment]
I noticed that some seleP songs are not marked as FA.

https://osu.ppy.sh/beatmapsets/41686
https://osu.ppy.sh/beatmapsets/46623
https://osu.ppy.sh/beatmapsets/20201

Those maps use vocaloid name as artist, `seleP` is provided in source field. Those 3 songs are in seleP's FA page.

![image](https://user-images.githubusercontent.com/5307179/189146424-e8c8f116-4cf9-44cf-bdd6-458000f5b751.png)

**Comment by JBHyperion** [comment]
Some Omoi tracks brought to my attention:

Teioh Education - matching FA listing
[osu](https://osu.ppy.sh/beatmapsets/1278333) | [taiko](https://osu.ppy.sh/beatmapsets/1466141)

Snow Drive(01.23)
[osu](https://osu.ppy.sh/beatmapsets/478405) - FA listing has an additional space
[loved](https://osu.ppy.sh/beatmapsets/379783) - seems to match FA listing

Totsugeki Zenya no Dance - FA listing uses an English translation, "Dance on the Eve of Attack" 
[osu](https://osu.ppy.sh/beatmapsets/342030)

Kiseki sae mo - FA listing omits a space between "sae" and "mo"
[osu](https://osu.ppy.sh/beatmapsets/1322144) | [taiko](https://osu.ppy.sh/beatmapsets/1357597)


Additionally, still some [outstanding MIMI tracks I brought up before](https://github.com/ppy/osu-web/issues/8111#issuecomment-920348561) that may have been missed.

**Comment by pishifat** [comment]
tsunamix's songs use a different alias, so none of their ranked maps are marked 
https://osu.ppy.sh/beatmapsets?q=tsunamix_underground
https://osu.ppy.sh/beatmaps/artists/329

and this shouldn't be marked https://osu.ppy.sh/beatmapsets/1414510#taiko/2915513

**Comment by peppy** [comment]
I've fixed tsunamix. Other fixes will come a bit later.

**Comment by Shmiklak** [comment]
https://osu.ppy.sh/beatmapsets/1910029#osu/3939705

This is a Monstercat song which didn't receive its not Featured Artist marker nor Monstercat release marker in game

**Comment by pishifat** [comment]
everything under "Songs" on the monstercat listing (https://osu.ppy.sh/beatmaps/artists/255) isn't being tagged for whatever reason (including the map commented above)
some other examples:
https://osu.ppy.sh/beatmapsets/1885930#osu/3883169
https://osu.ppy.sh/beatmapsets/1888841#osu/3890044
https://osu.ppy.sh/beatmapsets/1905153#osu/3928422

**Comment by op45667** [comment]
https://osu.ppy.sh/beatmapsets/1897902#taiko/3911679

My map, which was ranked back in December, still hasn't had a featured artist tag added after a minor change (adding a space after a colon for standardization reasons). This is the first ranked map of this song that contains that space, which probably screwed with the FA tagging system.

**Comment by Susko3** [comment]
https://osu.ppy.sh/beatmapsets/114480
https://osu.ppy.sh/beatmapsets/100016

Artist is listed as `Circus-P` instead of [`CircusP`](https://osu.ppy.sh/beatmaps/artists/78).

**Comment by h3oCharles** [comment]
i have some maps for grave, not sure what happened

https://osu.ppy.sh/beatmapsets/1966374 little drummer girl
https://osu.ppy.sh/beatmapsets/1966366 rise above
https://osu.ppy.sh/beatmapsets/1966360 words i never said in dnb

**Comment by aryoadhi** [comment]
Have another beatmap with proper artist name and title, but unmarked for unknown reason.

https://osu.ppy.sh/beatmapsets/1973673 Rabbit House - Gehenna

**Comment by marcostudios** [comment]
As https://osu.ppy.sh/community/forums/topics/1782291?n=1 said https://osu.ppy.sh/beatmapsets/41686#osu/131564 isn't marked even though [artist got featured](https://osu.ppy.sh/home/news/2021-02-10-new-featured-artist-selep)

**Comment by pishifat** [comment]
https://osu.ppy.sh/beatmapsets/31347#osu/103054
https://osu.ppy.sh/beatmapsets/1820561#osu/3735734
some false positives that shouldnt be marked as fa

**Comment by QTPC** [comment]
missing fa marker:
https://osu.ppy.sh/beatmapsets/1074464
https://osu.ppy.sh/beatmapsets/891345
This song is not in the KRUX alias you can see it [here](https://manbow.nothing.sh/event/event.cgi?action=More_def&num=95&event=110), so please change it

**Comment by yasuhosu** [comment]
hi, missing FA marker here:
https://osu.ppy.sh/beatmapsets/1729416#taiko/3534465
This is caused by the template being wrong. instead of `Shizukana Yoru ni` it is `Shizuka na Yoru ni` (the `na` particle is currently not standing alone)
There is [this](https://osu.ppy.sh/beatmapsets/1370680#osu/2834401) ranked set currently which uses the same title.

**Comment by pishifat** [comment]
https://osu.ppy.sh/beatmapsets/1885318
another false positive that shouldnt be marked as fa

**Comment by SoundBoomer** [comment]
https://osu.ppy.sh/beatmapsets/96580#osu/258816
https://osu.ppy.sh/beatmapsets/2034676#osu/4242178
https://osu.ppy.sh/beatmapsets/1742714#osu/3562609
A bunch of false positives tied to the A? (Aquestion) featured artist

**Comment by Wrongwayscout** [comment]
Missing FA marker/incorrect FA listing metadata:
https://osu.ppy.sh/beatmapsets/1428384

Map metadata spells title `Veracila` (with a c) while featured artist listing spells it `Verasila` (with an s)

Metadata sources linked in map discussion:
https://osu.ppy.sh/beatmapsets/1428384/discussion/-/generalAll#/2280817

**Comment by Carpihat** [comment]
https://osu.ppy.sh/beatmapsets/682598#taiko/1443524
https://osu.ppy.sh/beatmapsets/664795#taiko/1456026
https://osu.ppy.sh/beatmapsets/682732#taiko/1443785
https://osu.ppy.sh/beatmapsets/2069976#mania/4331264

Missing FA mark because the title in FA listing is wrong.

**Comment by h3oCharles** [comment]
https://osu.ppy.sh/beatmapsets/2079507 dreaming from twin trials contest has mismatching metadata

**Comment by Shompi** [comment]
https://osu.ppy.sh/beatmapsets/2024359 False positive **marked** as FA

**Comment by cl8n** [comment]
^ same with <https://osu.ppy.sh/beatmapsets/96580>
ice cream too strong

**Comment by ShiiTsuin** [comment]
https://osu.ppy.sh/beatmapsets/2116675#osu/4448144
Shouldn't be marked as FA, posted on behalf of AirinCat

**Comment by pishifat** [comment]
missing the fa marker https://osu.ppy.sh/beatmapsets/1972749#osu/4093503
yet https://osu.ppy.sh/beatmapsets/1971987#osu/4091819 has it with the same metadata

the template osz was updated after the first map was ranked, so that may be related

**Comment by Ripplechu** [comment]
https://osu.ppy.sh/beatmapsets/2155736#osu/4544081 unmarked Fa tag

**Comment by Shmiklak** [comment]
https://osu.ppy.sh/beatmapsets/1885318#osu/4127309 seems to be false positive as the song isn't available under FA listing

**Comment by pishifat** [comment]
https://osu.ppy.sh/beatmapsets/2079919#osu/4520026 another false positive (artist has a licensed song called "Moai")

**Comment by mcendu** [comment]
https://osu.ppy.sh/beatmapsets/1151089 unmarked, linked in DGK showcase

**Comment by R-Trigger** [comment]
https://osu.ppy.sh/beatmapsets/2162219#mania/4559801 The song “rabbit in the black room”is in the fa listing, but there is a remaster ver called “rabbit in the black room(2023 VIP), and it has fa marked in the information page

**Comment by ikin5050** [comment]
https://osu.ppy.sh/beatmapsets/2199195#taiko/4662115 falsely marked as FA because of Silentroom - Angel Echo existing

**Comment by Dropinx** [comment]
https://osu.ppy.sh/beatmapsets/2203526#taiko/4664066 Missing FA marker, but https://osu.ppy.sh/beatmapsets/1929063#mania/3984270 has it with the same metadata.

**Comment by tilda** [comment]
https://osu.ppy.sh/beatmapsets/2195640#osu/4645958

**Comment by MolybdenumGaming** [comment]
I've noticed recently that due to the Ranking Criteria's standardization for metadata, certain tracks do not obtain the Featured Artist website tag like they should.

A example of this (which was recently fixed) is "ELFENSJoN - Hyousou wa Hakuen o Matoite", which was originally "ELFENSJoN - Hyousou wa Hakuen wo Matoite" (Ranking Criteria standardizes "wo" to "o" in romanisation)

Another example has popped up, being "uma vs. Morimori Atsushi - Re:End of a Dream" and "uma vs. Morimori Atsushi - Re:End of a Dream (long ver.)" which Ranking Criteria standardizes to "uma vs. Morimori Atsushi - Re: End of a Dream (long ver.)" and "uma vs. Morimori Atsushi - Re: End of a Dream".

**Comment by Muchin708** [comment]
https://osu.ppy.sh/beatmapsets/421554#osu/4303937 
seems to be a false positive as it is totally a different version to the one in the fa listing

**Comment by O1CDeif** [comment]
https://osu.ppy.sh/beatmapsets/1175640#osu/2452076
https://osu.ppy.sh/beatmapsets/2180222#fruits/4605320
FA listing has the full version for the download link, which should've been called "Kanata ni Mau wa Sakura no Senritsu / 彼方に舞うは桜の旋律". However the FA listing got it named as "Kanata ni Mau wa Sakura no Shirabe / 彼方に舞うは桜の律べ", which is indeed the shorter version of the same song made for 初音ミク Project DIVA Arcade.
- Longer Version: https://www.nicovideo.jp/watch/sm18787484
- Shorter Version: https://www.nicovideo.jp/watch/sm21535527

**Comment by Ripplechu** [comment]
https://osu.ppy.sh/beatmapsets/2214653#taiko/4692536 N.M.W KINEMA106 unmarked Fa marker

**Comment by osuZiRoX** [comment]
https://osu.ppy.sh/beatmapsets/31347
https://osu.ppy.sh/beatmapsets/2059533
https://osu.ppy.sh/beatmapsets/1938524
https://osu.ppy.sh/beatmapsets/1820561
https://osu.ppy.sh/beatmapsets/106320

All seem to be incorrectly marked

**Comment by SoundBoomer** [comment]
https://osu.ppy.sh/beatmapsets/1873236#osu/3854548 Incorrect artist marked as "A?" featured artist

**Comment by 4lwl4** [comment]
https://osu.ppy.sh/beatmapsets/1795159
no longer has an fa marker after fixing the romanized title to follow standardization

**Comment by Glinte** [comment]
The following two tracks seem to be incorrectly translated in the [FA page](https://osu.ppy.sh/beatmaps/artists/31). Doesn't seem to be a issue with the "feat." part because "Camellia feat. Nanahira - Kizuitara Shunkashuutou" are marked correctly.

Mapset name: Camellia feat. Nanahira - Energy * Drin-ko Fein-chan!
FA page track name: Camellia feat. Nanahira - Energy Doll Ko * Fuin-chan
https://osu.ppy.sh/beatmapsets/257407

Mapsets name: Camellia feat. Nanahira - Energy * Drin-ko Fein-chan! (Camellia's "MONSTERISTIC" D. M. P. Remix)
FA page track name: Camellia - Energy * Drinking Musume * Fein-chan! (Camellia's "MONSTERISTIC" D. M. P. Remix)
https://osu.ppy.sh/beatmapsets/1316055
https://osu.ppy.sh/beatmapsets/883595
https://osu.ppy.sh/beatmapsets/1820163
https://osu.ppy.sh/beatmapsets/1672652

**Comment by Susko3** [comment]
(quotes from https://github.com/ppy/osu-web/issues/8111#issuecomment-1019765428)

> I guess it does depend on how we use the featured artist badge going forward, and whether having these maps marked as such leads to further confusion.

When https://github.com/ppy/osu/pull/31238 goes in, the featured artist badge is going to be more important.

> While they aren't 100% correct, I'm not sure these edge cases matter too much. **They are harder ones to automate** because the titles (like "feat. someone") for a cover are not standard.

The [ranking criteria](https://osu.ppy.sh/wiki/en/Ranking_criteria/Simplified_ranking_criteria#tags) require FA maps to be marked with `featured artist` tag, maybe using this to give/revoke the featured artist badge is less-error prone. The automation can still be used for linking back to the original song.

How hard is it to change tags on already uploaded beatmapsets?

**Comment by Hiviexd** [comment]
> The [ranking criteria](https://osu.ppy.sh/wiki/en/Ranking_criteria/Simplified_ranking_criteria#tags) require FA maps to be marked with featured artist tag, maybe using this to give/revoke the featured artist badge is less-error prone. The automation can still be used for linking back to the original song.

Do note that for a long time the `featured artist` tag was being used for non-licensed songs made by FAs (which shouldn't have the website FA tag applied to them) until this [RC proposal](https://osu.ppy.sh/community/forums/topics/1801856?n=1) got [merged](https://github.com/ppy/osu-wiki/pull/10175), so I wouldn't consider that as a reliable metric.

**Comment by ziv-vy** [comment]
These songs are not licensed. They are just as dangerous to host as any other mp3. I think they should be untagged.

**Comment by Repflez** [comment]
https://osu.ppy.sh/beatmapsets/2390396#osu/5174234 this map is labeled as FA despite kanye west not being FA, clicking the FA label sends to <https://osu.ppy.sh/beatmaps/artists/tracks/4880> (T & Sugah)

**Comment by Walavouchey** [comment]
some maps of fa songs that have differing artist fields:
https://osu.ppy.sh/beatmapsets/41686 ("Lily" (the vocaloid used) instead of "seleP")
https://osu.ppy.sh/beatmapsets/1376486 ("Risshuu" (old name) instead of "Rish")
https://osu.ppy.sh/beatmapsets/529724 ("kalon." (the song's vocalist) instead of "KINEMA106")

**Comment by peppy** [comment]
> some maps of fa songs that have differing artist fields: https://osu.ppy.sh/beatmapsets/41686 ("Lily" (the vocaloid used) instead of "seleP") https://osu.ppy.sh/beatmapsets/1376486 ("Risshuu" (old name) instead of "Rish") https://osu.ppy.sh/beatmapsets/529724 ("kalon." (the song's vocalist) instead of "KINEMA106")

There still needs to be some work done to improve the automation of this, but to fix edge cases like this, it may benefit from manual marking. Would an ASS command be something that would help here? cc @Walavouchey @Hiviexd

**Comment by Hiviexd** [comment]
Yes, that would be great. Ability to manual mark the FA tag has been requested multiple times in the past IIRC.

**Comment by peppy** [comment]
The caveat is that unmarking would not be supported (we'd be looking to fix the automatic rules for that instead).

**Comment by Walavouchey** [comment]
there's probably a low volume of such cases. probably works best if we take reports from only here and only manually mark older maps we know aren't caught in the automatic process

**Comment by antegaming432-creator** [comment]
https://osu.ppy.sh/beatmapsets/2305742#osu/4929890
https://osu.ppy.sh/beatmapsets/1024635#osu/2645456

Got Me Thinking (feat. Veela) - Maduk

This specific song is not on the featured artist listing, the marker should be removed.
The set in graveyard will be ranked as it has 2 bns

**Comment by pishifat** [comment]
maps with metadata matching anything in the "Songs" section of https://osu.ppy.sh/beatmaps/artists/255 are missing the marker

**Comment by Walavouchey** [comment]
https://osu.ppy.sh/beatmapsets/869649#osu/1819587 (https://osu.ppy.sh/beatmaps/artists/297) "-45 - G e n g a o z o" (map uses "455-38B" as the artist and appends " - f o o n" to the title)

[reported on reddit](https://www.reddit.com/r/osugame/comments/1stq2j7/fa_tag_on_beatmap/)

currently ranked edge cases (ignore remixes/edits) https://osu.ppy.sh/beatmapsets?q=title%3D%22G%20e%20n%20g%20a%20o%20z%20o%22. 455-38B probably needs to be added as an alias

**Comment by pishifat** [comment]
https://osu.ppy.sh/beatmapsets/789849 false positive, doesn't match any metadata from associated track `680` (Camellia - crystallized)

**Comment by UristMcKobold** [comment]
https://osu.ppy.sh/beatmapsets/236396#osu/717133
https://osu.ppy.sh/beatmapsets/322808#osu/717632
https://osu.ppy.sh/beatmapsets/528694#fruits/1121287
Artist metadata is a vocalist, which is why I assume it wasn't marked.

**Comment by VulkinLove** [comment]
https://osu.ppy.sh/beatmapsets/980047 False positive. The FA version only has the instrumental TV Size, formerly known as (osu! edit), and the vocal version is not on the listing.

**Comment by Susko3** [comment]
[Kuba Oms - My Love (Redside Remix) (feat. Hatsune Miku)](https://osu.ppy.sh/beatmapsets/1388993) matched to Kuba Oms (https://osu.ppy.sh/beatmaps/artists/tracks/243) instead of Redside (https://osu.ppy.sh/beatmaps/artists/tracks/4836)

**Comment by Muchin708** [comment]
https://osu.ppy.sh/beatmapsets/2613191#osu/5854437 false positive as this version isn't licensed by the artist
